From: Jan Engelhardt <jengelh@inai.de>
Date: 2012-05-11 08:34:41 +0200

Remove build time references so build-compare can do its work.
---
 librecad/src/main/qc_applicationwindow.cpp |    1 -
 1 file changed, 1 deletion(-)

--- a/librecad/src/main/qc_applicationwindow.cpp
+++ b/librecad/src/main/qc_applicationwindow.cpp
@@ -2380,7 +2380,6 @@ void QC_ApplicationWindow::showAboutWind
         #elif defined(Q_CC_MSVC)
             tr("Compiler: Microsoft Visual C++") + "\n" +
         #endif
-        tr("Compiled on: %1").arg(__DATE__) + "\n" +
         tr("Qt Version: %1").arg(qVersion()) + "\n" +
         tr("Boost Version: %1.%2.%3").arg(BOOST_VERSION / 100000).arg(BOOST_VERSION / 100 % 1000).arg(BOOST_VERSION % 100)
     );
